andela/nightcrawler-backend
# #167727317 create notification for new travel request

#### What does this PR do?

- Create notification for new travel request

#### Description of Task to be completed?

- Emit a notification via socket.io when a requester makes a trip request.

- Send an email to both requester and manager when a trip request is made.

- Create a get all notifications endpoint ```api/v1/notifications```  incase the user wasn't online when the notification was sent.

#### How should this be manually tested?

- Clone the repository and cd into it.

- Run ```npm start dev``` to start the server 

- Post a trip request, and you should receive an email or in app notification.

- Also hitting the get notifications endpoint should return all notifications reference documentation.

#### Any background context you want to provide?

- 

#### What are the relevant pivotal tracker stories?

- [167727317](https://www.pivotaltracker.com/story/show/167727317)

#### Questions:
